src/i18n/languages/zh-CN.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * @preserve
+ * Authors: Phyllis Yen
+ * Last updated: Mar 9, 2018
+ *
+ * Description: Definition file for Traditional Chinese - Taiwan language-country.
```

Comment:
Please change this line (language-country should correspond to the proper one, chosen from https://docs.microsoft.com/en-us/previous-versions/commerce-server/ee825488(v=cs.20))